src/components/UserProfile.jsx

```diff
@@ -1,15 +1,51 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import '../styles/index.scss';
 import ninja from '../assets/images/ninja.jpg';
+import { increment, decrement } from '../actions';
 
 const UserName = () => <h2>Forsetti</h2>;
 
 const Avatar = () => <img src={ninja} alt="ninja"/>;
 
-const Userprofile = () => 
-    <div>
-        <UserName />
-        <Avatar />,
-    </div>
+class Userprofile  extends Component{
```

Comment:
Good job @Chiazokam, based on the convention we are to use as instructed by @paradisekelechi, the file naming should be the same with the class name, and both need to follow Pascal naming convention. 

Going by this, the name of the file is `UserProfile.js`, the class name should be `UserProfile`

Reply:
Done